charts/orchestrator/Chart.yaml

```diff
@@ -1,23 +1,3 @@
-#################################################################################
-# Copyright (c) 2021,2022 Catena-X
```

Comment:
why did you remove this header? besids referencing catena-x, the header looked fine?

Reply:
Added the correct header